docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Decentraland's catalyst
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+1. If you've added code that should be tested, add tests.
+1. Before changing the API, update the protocol by sending a pull request at the [api spec repository](https://github.com/decentraland/catalyst-api-specs).
```

Comment:
I would change this point as even if anyone wants or considers that the API should change, even if right, we need to follow a process, file an ADR, plan the change and communicate. 
I guess that I would add something like 
1. Contribution should not break the current protocol implementation, this means not breaking any existing API, as this could have an impact on many people work and API consumers. If your contribution needs to add a breaking change, please file a [DAO poll](https://governance.decentraland.org/) so that the community can understand and accept your suggestion.